setup.cfg

```
@@ -1,7 +1,7 @@
 [build_sphinx]
 source-dir = docs
```

Should this be `doc` too?
```suggestion
source-dir = doc
```